core/templates/dev/head/pages/exploration_editor/ExplorationSaveAndPublishButtons.js

```diff
@@ -13,98 +14,520 @@
 // limitations under the License.
 
 /**
- * @fileoverview Controller for the exploration save & publish buttons.
+<<<<<<< HEAD
```

Comment:
@seanlip do you have any suggestions for dropping my changes for explorationsaveandpublish buttons? I'm guessing just update the .js file back to @Keksike changes.